libs/CMakeLists.txt

```diff
@@ -359,6 +359,10 @@ if(HPX_WITH_DATAPAR_VC)
   target_link_libraries(hpx_core PUBLIC Vc::vc)
 endif()
 
+if(HPX_WITH_DATAPAR_EVE)
```

Comment:
`HPX_WITH_DATAPAR_EVE` is defined internally when user passes this flag `-DHPX_WITH_DATAPAR_BACKEND=DATAPAR_EVE`.

Reply:
Wouldn't it be sufficient to do `if("${HPX_WITH_DATAPAR_BACKEND}" STREQUAL "DATAPAR_EVE")` instead of introducing yet another cmake options that duplicates existing functionalities?